workspaces/editors.md

```diff
@@ -178,3 +179,37 @@ participants. If they do, click **Accept** to proceed.
 
 At this point, you'll be able to share your project and work with your partner
 in real-time.
+
+## RStudio
+
+Coder supports [RStudio](rstudio.com). To create a workspace that lets you use
+RStudio:
+
+1. Create an image that includes RStudio (see
+   [rocker/rstudio](https://hub.docker.com/r/rocker/rstudio) for an example).
+
+1. [Create a workspace](getting-started.md#2-create-a-workspace) using the image
+   you created in the previous step.
+
+1. **Optional.** RStudio requires authentication with a Linux user and stores
+   all files in their home directory. If, from your Docker image, you're using the
+   root user, `coder`, or a user that doesn't have a home directory
+   or known password, then we recommend creating a new user for your RStudio
+   projects. To do so, run:
+
+   ```console
+   useradd --create-home myuser     # replace myuser with your username
+   passwd myuser     # provide and confirm your chosen password when prompted
+   ```
+
+1. At this point, you can go to **Applications** to launch RStudio:
+
+   ![Applications with RStudio launcher](../assets/workspaces/rstudio.png)
+
+   Sign in using your RStudio user credentials (RStudio auto-populates
+   the fields your Coder credentials, so you'll need to change these to the ones
+   you just created).
+
+   > All RStudio user data is stored in the `home` directory associated with this
+   > user, not your Coder user (that is, you'll find it in `/home/myuser` where
```

Comment:
I think the concept of "your Coder user" isn't obviously clear: the reason we and many are set to a user `coder` is because we in our images (https://github.com/cdr/enterprise-images) set the `USER` to `coder`:

https://github.com/cdr/enterprise-images/blob/11926804262ead6c2ac41061488ddcd043868216/images/node/Dockerfile.ubuntu#L16

There's nothing inherently tied to the user of `coder` otherwise; if I homebrew an image like so:

```dockerfile
FROM node:latest
USER myuser
```

Then my home directory will be `/home/myuser`.

As a result - I'm not entirely sure of the best wording, but I'd want to refrain from "your Coder user" to avoid confusion in cases whereby folks are using a custom non-Coder image.

Maybe something like:

> All RStudio data is stored in the home directory associated with this user, which may be different than the user on workspace startup (that is, you'll find the data in `/home/myuser` where `myuser` is your RStudio username).